iris/src/iris/pipeline/shared/citation_pipeline.py

```
@@ -82,62 +90,186 @@ def create_formatted_lecture_string(
+        except json.JSONDecodeError:
+            return "", ""
+        keyword = self._sanitize_citation_field(str(data.get("keyword", "")).strip())
+        summary = self._sanitize_citation_field(str(data.get("summary", "")).strip())
+        return keyword, summary
+
+    def _build_keyword_summary_map(
+        self,
+        pipeline,
+        language_instruction: str,
+        used_numbers: list[int],
+        user_language: str,
```

Ruff (0.14.14)

[warning] Unused method argument: `user_language`

(ARG002)